### ISSUE TITLE
Cannot send XSN from Trezor Device

### ISSUE THREAD

**Issue by mat888** [X9Developers/block-explorer#207]
Going through all the normal steps of sending XSN from Trezor, with every step appearing to work as intended up to and including the final confirmations on the device, but the TX is never actually sent.

### Expected behavior

XSN transactions to be sent from the  Trezor, XSN web interface.

### Actual behavior

All intermediate  steps appear to work, no errors appear, but once finished the transaction is not deducted from the wallet nor sent at all. One single time, after unplugging the wallet and plugging it back in and repeating these steps above the TX went through immediately, but this method never reproduced a successful transaction and all previous and future attempts suffered the same fate as the previously mentioned failed TX.

### Steps to reproduce the behavior
This happened in Firefox and Google Chome on a Trezor One.
Plug Trezor device in, log in via official Trezor wallet site. Open Trezor Tab in XSN block explorer, input address intending to send to and amount, hit send, re-enter credentials and then finally confirm the transaction with prompts on the Trezor Device itself.

**Comment by fkfkfkdsds** [comment]
Might it be, that you tried to send a bigger ammount of XSN?
If so try the new transaction ammount "Very high" and have a look if it helps.

**Comment by AlexITC** [comment]
Indeed, there is a issue that when dealing with lots of UTXOs, the fee amount may not be enough, right now its computed in a naive way instead of estimating how much is required.

The common way to solve this for now, is to send smaller amounts, or choose the highest fee.

**Comment by mat888** [comment]
Seems to be very specific the OS. Works without a hitch on an Arch distro. KUbuntu (latest build) is where this issue persisted. Could be a weird configuration or something else related to Trezor - I don't want this to be too much of a time sink or worry.

**Comment by AlexITC** [comment]
I don't know if you are hitting a different issue, in this case, we detected that the fees are being hardcoded instead of taking them as the wallet does (we take 1000 satoshis instead of 1000 satoshis/kb), when the transaction gets big enough, the network rejects it due the fee being not enough.

This will be fixed.

**Comment by trancilo** [comment]
I experienced the same issue just now. Trying to send 10, 30 and 50 xsn from my trezor to another address. With a fee of 100satoshis
All three the transactions failed without error message or transaction ID
I **was** able to send all three of the amounts when selecting  500 satoshis.

@AlexITC what do you mean with "when the transaction gets big enough" 
I don't view 10xsn as a large transaction? Or are you talking about transactions per minute on the network?

[edit]
I noticed that the actual transaction fee shown on xsnexplorer.io is 0.000005
And I just now realized that we are talking XSN satoshis. Not BTC satoshis. Makes sense

**Comment by AlexITC** [comment]
> "when the transaction gets big enough"

The amount is irrelevant but the number of outputs required for the transaction is what get us the need for a higher fee, ideally, the integration should compute the fee automatically like xsn-qt does but we haven't fixed that.
